cartography/intel/gcp/__init__.py

```diff
@@ -0,0 +1,41 @@
+from oauth2client.client import GoogleCredentials, ApplicationDefaultCredentialsError
+import logging
+
+from cartography.intel.gcp import crm
+# TODO: use these for analysis job
+# from cartography.util import run_analysis_job, run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+
+def start_gcp_ingestion(session, config):
+    common_job_parameters = {
+        "UPDATE_TAG": config.update_tag,
+    }
+    try:
+        # Explicitly use Application Default Credentials.
+        # See https://oauth2client.readthedocs.io/en/latest/source/
+        #             oauth2client.client.html#oauth2client.client.OAuth2Credentials
+        credentials = GoogleCredentials.get_application_default()
+    except ApplicationDefaultCredentialsError as e:
+        logger.debug("Error occurred calling GoogleCredentials.get_application_default().", exc_info=True)
+        logger.error(
+            (
+                "Unable to initialize Google Compute Platform creds. If you don't have GCP data or don't want to load "
+                "GCP data then you can ignore this message. Otherwise, the error code is: %s "
+                "Make sure your GCP credentials are configured correctly, your credentials file (if any) is valid, and "
+                "that the identity you are authenticating to has the securityReviewer role attached."
+            ),
+            e
+        )
+        return
+    crm.sync_organizations(session, credentials, config.update_tag, common_job_parameters)
+
+    """
+    # TODO - add internet exposure
+    run_analysis_job(
+        'gcp_compute_asset_exposure.json',
+        session,
+        common_job_parameters
+    )
+    """
```

Comment:
Either implement or remove from this PR (applies to the TODO in the imports block as well).